README.md

```diff
@@ -1 +1,50 @@
 # authboot
+
+A simple [`slay`][slay] preboot that initializes a namespaced `lookup` function
+and `middeware` on the app object to be used within the application. Its meant
+to wrap up some configuration conventions to be more easily reused in multiple
+slay applications without duplicating code unnecessarily.
+
+
+## install
+`npm install authboot --save`
+
+## usage
+
+```js
+// root slay preboot in preboots/index.js or preboots.js
+
+module.exports = function (app, opts, callback) {
+  app.preboot(require('authboot')({
+    users: {
+      name: 'password'
```

Comment:
Are we expecting plaintext passwords here?

Reply:
Are there plans to support a non-basic-auth auth scheme here?

Reply:
@msluther the idea for the short term is for passwords to be stored in an encrypted config that is decrypted on startup. We are leveraging basic auth as the means of using actual API keys where its just useful to have a username to identify the team and to also continue support for our `npm` case more easily. The part where this is actually more secure comes from how the passwords are generated and stored.

The lookup function can be configured in the future to use some kind of more secure async store.

Reply:
Even though the passwords are expected to be stored in encryption at rest, it's still worthwhile to stored them salted & encrypted.